docs/src/gs/create-workload-cluster.md

```diff
@@ -8,15 +8,19 @@ Choose one of the available templates for to create your workload clusters from
+|-------------------------------------------|---------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `OCI_COMPARTMENT_ID`                      |                     | The OCID of the compartment where the OCI resources are to be created                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| `OCI_IMAGE_ID`                            |                     | The OCID of the Compute Image (Oracle Linux or <br/>Ubuntu) with which to create the Kubernetes nodes. This same image is used for both the control plane and the worker nodes. Before picking an image look at the capabilities of the image.  Some images only support certain shapes, but most official images should support most (if not all) shapes. https://docs.oracle.com/en-us/iaas/api/#/en/iaas/20160918/Shape/ListShapes                                                 |
+| `OCI_CONTROL_PLANE_SHAPE`                 | VM.Standard.E4.Flex | The shape of the Kubernetes nodes                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| `OCI_CONTROL_PLANE_SHAPE_MEMORY_IN_GBS`   |                     | The amount of memory to be allocated to the instances. If not provided it is automatically computed by compute API.                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
Please replace with "The amount of memory in GBs to be allocated to the control plane instances. If not provided, the memory is automatically computed by compute API."